packages/frontend/src/Build.svelte

```diff
@@ -193,6 +194,7 @@ async function buildBootcImage() {
     type: buildType,
     arch: buildArch,
     filesystem: buildFilesystem,
+    chown: buildChown,
```

Comment:
What do you think about having a default value to change ownership to the current user? I know that's not the 'bootc default', but I think in our context people are more likely to hit the issue like Axel did.

Reply:
I agree and it would be a good idea to implement. Especially being on Linux, users will encounter this issue a lot.

I'll add this in a new commit!